data/Agrim-Sharma174.json

```
@@ -0,0 +1,27 @@
+{
+	"name": "Agrim",
+	"type": "personal",
+	"bio": "Smart Contract Developer | Web Developer | Technical Writer",
+	"links": [{
+			"name": "Read my Blogs",
+			"url": "https://agrimsharma.hashnode.dev/",
+			"icon": "FaHashnode"
```

You're probably be using an `invalid` icon. It should be `SiHashnode`

```suggestion
			"icon": "SiHashnode"
```